buildpipeline/pipeline.json

```diff
@@ -283,6 +311,75 @@
           }
         },
         {
+          "Name": "DotNet-CoreFx-Trusted-Windows",
+          "Parameters": {
+            "PB_Platform": "arm",
+            "PB_BuildArguments": "-buildArch=arm -Release -portable -- /p:SignType=real /p:RuntimeOS=win10",
```

Comment:
I know you are just following the existing pattern but is probably better to use `$(PB_Platform)` in these other parameters (assuming that works).